src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -5214,6 +5214,27 @@ protected sealed override ImmutableArray<ISymbol> GetDeclaredSymbolsCore(SyntaxN
 
             return null;
         }
+
+#pragma warning disable RSEXPERIMENTAL002 // Internal usage of experimental API
+        public InterceptableLocation? GetInterceptableLocation(InvocationExpressionSyntax node)
+        {
+            CheckSyntaxNode(node);
+            if (node.GetInterceptableNameSyntax() is not { } nameSyntax)
+            {
+                return null;
+            }
+
+            var tree = node.SyntaxTree;
+            var text = tree.GetText();
```

Comment:
Take and pass along cancellation token

Reply:
I made an assumption that when the syntax is loaded, the text is also loaded. Was I wrong about that?

Reply:
yes (which is also why that method takes a cancellataion token).  The text can be dumped out of main memory (so it can be reconstituted on hte fly as needed).

Reply:
Done